src/pages/Pokemon/pokemon.css

```diff
@@ -1,102 +1,59 @@
+body {
```

Comment:
mate I'd say let's not style body cos it will affect all the pages and cause CHAOS mate. Better to only use specific calssNames to prevent styles spilling over into all the files